cgmes/cgmes-model/src/main/java/com/powsybl/cgmes/model/triplestore/CgmesModelTripleStore.java

```diff
@@ -257,6 +257,11 @@ public PropertyBags fullModel(String cgmesProfile) {
         return namedQuery("fullModel", cgmesProfile);
```

Comment:
Should the "fullModel" query not be deleted? I think it will not be used anymore.

Reply:
yes, it has been removed in the last commit